templates/e-commerce/partials/hero/hero.snip.html

```diff
@@ -0,0 +1,20 @@
+{{#hero}}
+  <div class="e-commerce-ampstart-hero center">
+    <div class="e-commerce-ampstart-hero-img">
```

Comment:
you should not need this wrapping div , you can add this class directly to the image